cumulusci/tasks/salesforce/update_dependencies.py

```diff
@@ -56,6 +56,7 @@ class UpdateDependencies(BaseSalesforceTask):
         "packages_only": {
             "description": "Install only packaged dependencies. Ignore all unmanaged metadata. Defaults to False."
         },
+        **PACKAGE_INSTALL_TASK_OPTIONS,
```

Comment:
Do we really want to include password here?

Reply:
Probably not. Correcting.